cmd/build/v2/build_test.go

```diff
@@ -158,15 +159,14 @@ func (a *fakeAnalyticsTracker) TrackImageBuild(meta ...*analytics.ImageBuildMeta
 	a.metaPayload = meta
 }
 
-func NewFakeBuilder(builder OktetoBuilderInterface, registry oktetoRegistryInterface, cfg oktetoBuilderConfigInterface, analyticsTracker analyticsTrackerInterface) *OktetoBuilder {
+func NewFakeBuilder(builder buildCmd.OktetoBuilderInterface, registry oktetoRegistryInterface, cfg oktetoBuilderConfigInterface, analyticsTracker analyticsTrackerInterface) *OktetoBuilder {
 	return &OktetoBuilder{
-		Registry:          registry,
-		Builder:           builder,
+		Registry: registry,
+		// Builder:           builder,
```

Comment:
```suggestion
```

Should we remvoe this?

Reply:
Yes, good catch!

Reply:
Removed: https://github.com/okteto/okteto/pull/4189/commits/ffb513fbeb752fe766c03aea2cd3e0090764ac94